spec/fixtures/files/datahashes/core/media1.json

```diff
@@ -8,14 +8,15 @@
     "measuredPart": "framed",
     "dimensionSummary": "Past is gone",
-    "dimension": "base;weight;;circumference",
-    "measuredByPerson": "Gomongo;;Comodore;",
-    "measuredByOrganization": "Cuckoo;",
```

Comment:
If you check your CollectionSpace record and the XML file, you can see I moved the lines around, so the "Measured by" order is: Gomongo, Comodore, Cuckoo, (blank). 

This is related to a general limitation of the mapper that I am working to document for end users: 

If a field can be populated by multiple authorities (say Person and Organization), all the Person terms will be handled together and all the Organization terms will be handled together. 

This means the mapper can't really handle something like: dimension with person measurer, dimension with organization measurer, dimension with person measurer.